src/tests.js

```diff
@@ -11,7 +11,7 @@ const series = require('async/series')
 const parallel = require('async/parallel')
 const map = require('async/map')
 const each = require('async/each')
-const crypto = require('libp2p-crypto')
+const randomBytes = require('libp2p-crypto/src/random-bytes')
```

Comment:
This is an antipattern. We should not require a file within a module because there are no guarantees in the contract of the module that the file will remain there.

Reply:
How about using the standard node `crypto.randomBytes()`.  It;s a test so KISS.

Reply:
Should also work in the browser, it needs a check to see if it should use WebCrypto or crypto.

Reply:
Opened 2 PR's crypto vs. webcrypto see summary https://github.com/ipfs/interface-datastore/issues/10#issuecomment-341893957

Reply:
Is this an antipattern as well?

```js
describe('mystore', () => {
  require('interface-datastore/src/tests')({
```
see https://github.com/ipfs/interface-datastore#testsuite

I do not know `libp2p-crypto` very well, but it seems to me to be more of a collection of useful utils, where you hardly ever use the whole api (like lodash) rather than 'do one thing' module